disnake/ext/tasks/__init__.py

```diff
@@ -73,31 +100,35 @@ def cancel(self) -> None:
         self.future.cancel()
 
 
-class Loop(Generic[LF]):
+class Loop(Generic[CoroP]):
     """A background task helper that abstracts the loop and reconnection logic for you.
 
     The main interface to create this is through :func:`loop`.
     """
 
     def __init__(
         self,
-        coro: LF,
-        seconds: float,
-        hours: float,
-        minutes: float,
-        time: Union[datetime.time, Sequence[datetime.time]],
-        count: Optional[int],
-        reconnect: bool,
-        loop: asyncio.AbstractEventLoop,
+        coro: Coro[CoroP],
+        seconds: float = 0,
+        minutes: float = 0,
+        hours: float = 0,
+        time: Union[datetime.time, Sequence[datetime.time]] = MISSING,
+        count: Optional[int] = None,
+        reconnect: bool = True,
+        loop: asyncio.AbstractEventLoop = MISSING,
     ) -> None:
-        self.coro: LF = coro
+        """
+        .. Note:
+            If you're going to overwrite __init__ arguments, make sure to redefine .clone too
+        """
```

Comment:
i don't think this appears on the documentation, the docstring for `Loop` needs to be updated with this instead.

Reply:
The intent was for this *not to* be visible on the Loop itself, and instead on code introspection; I don't think any class documents any info like that

Reply:
makes sense, but in that case the rst formatting is unused.

Reply:
So you suggest to remove the formatting? It will look kind of out of place when a "short description of the function" is actually a single note

Reply:
fair. if we're committing to rst we might as well make it valid.
```suggestion
        """
        .. note:
            If you overwrite ``__init__`` arguments, make sure to redefine .clone too.
        """
```